### PR TITLE
RHOAI-12065 link update

### PR DESCRIPTION
**Pull request by aduquett**
Updating links to use tree/main repo

### CODE REVIEW

File: modules/ref-supported-runtimes.adoc [hunk 1]
```diff
@@ -44,7 +44,7 @@ ifndef::upstream[]
...
 
-. The Caikit Standalone runtime is based on link:https://github.com/caikit/caikit-nlp/tree/main[Caikit NLP^]. To use this runtime, you must convert your models to the Caikit embeddings format. For an example, see link:https://github.com/markstur/caikit-embeddings/blob/df9c9bc93187c0a17cb66b86d609f2cd102be97d/demo/server/bootstrap_model.py[Bootstrap Model^].
+. The Caikit Standalone runtime is based on link:https://github.com/caikit/caikit-nlp/tree/main[Caikit NLP^]. To use this runtime, you must convert your models to the Caikit embeddings format. For an example, see link:https://github.com/caikit/caikit-nlp/blob/main/tests/modules/text_classification/test_sequence_classification.py[Tests for sequence classification model^].
```
Comment (christinaexyou) [inline modules/ref-supported-runtimes.adoc:47]: Replace text with `Tests for embeddings model` and link with: https://github.com/caikit/caikit-nlp/blob/main/tests/modules/text_embedding/test_embedding.py

File: modules/ref-supported-runtimes.adoc [hunk 2]
```diff
@@ -35,7 +35,7 @@ ifdef::upstream[]
...
```
Comment (christinaexyou) [inline modules/ref-supported-runtimes.adoc:38]: Replace text with Tests for embeddings model and link with: https://github.com/caikit/caikit-nlp/blob/main/tests/modules/text_embedding/test_embedding.py

File: modules/ref-supported-runtimes.adoc [hunk 3]
```diff
@@ -44,7 +44,7 @@ ifndef::upstream[]
...
 
-. The Caikit Standalone runtime is based on link:https://github.com/caikit/caikit-nlp/tree/main[Caikit NLP^]. To use this runtime, you must convert your models to the Caikit embeddings format. For an example, see link:https://github.com/markstur/caikit-embeddings/blob/df9c9bc93187c0a17cb66b86d609f2cd102be97d/demo/server/bootstrap_model.py[Bootstrap Model^].
+. The Caikit Standalone runtime is based on link:https://github.com/caikit/caikit-nlp/tree/main[Caikit NLP^]. To use this runtime, you must convert your models to the Caikit embeddings format. For an example, see link:https://github.com/caikit/caikit-nlp/blob/main/tests/modules/text_embedding/test_embedding.py[Tests for embeddings model^].
```
Comment (bredamc) [inline modules/ref-supported-runtimes.adoc:47]: Same comment as [above](https://github.com/opendatahub-io/opendatahub-documentation/pull/453/files#r1759063784).

**Review by bredamc — APPROVED** [review]
LGTM, thanks!

**Pull request merged**